src/components/ui/avatar-frame.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+
+interface AvatarFrameProps {
+  src: string;
+  alt: string;
+  width?: number;
+  showGrid?: boolean;
+  height?: number;
+}
+
+export default function AvatarFrame({
```

Comment:
@Imon-selise Since this isn’t a Shadcn component, you should move it from components/ui to components/core if it will be used across multiple features or modules. Otherwise, place it within the specific feature module, like features/email, if it’s only relevant to that feature.